pages/design-systems.njk

```diff
@@ -151,7 +151,7 @@
                 thumbnail_image="/images/systems/casestudies/Indigo.png",
                 title="Indigo Design System",
                 description="A system for the Universal Service Administrative Co.",
-                href="indigo.universalservice.org",
+                href="http://indigo.universalservice.org",
```

Comment:
Shouldn't this be `https`?

Reply:
All the others were http. I've now switched all the external links on this page to https after confirming each works.